### ISSUE TITLE
convert-to-oc4ids: Better documentation on some individual transfoms

### ISSUE THREAD

**Issue by kindly** [open-contracting/ocdskit#134]
Some transforms need clearer user documentation for them to make sense to users especially the optional ones.

For example optional transform `buyer_role` should state it is used as `publicAuthority` role if no `publicAuthority` exists.

**Comment by jpmckinney** [comment]
Yes, the [optional transforms](https://ocdskit.readthedocs.io/en/latest/cli/oc4ids.html#optional-transforms) should be clarified.

I'm not sure that the [other transforms](https://ocdskit.readthedocs.io/en/latest/cli/oc4ids.html#transforms) are clear either. If not, then we should just link to the [mapping documentation](https://standard.open-contracting.org/infrastructure/latest/en/cost/).
